core/src/main/scala/cats/data/XorT.scala

```diff
@@ -185,11 +186,10 @@ private[data] abstract class XorTInstances extends XorTInstances1 {
   implicit def xorTShow[F[_], L, R](implicit sh: Show[F[L Xor R]]): Show[XorT[F, L, R]] =
     functor.Contravariant[Show].contramap(sh)(_.value)
 
-  implicit def xorTBifunctor[F[_]](implicit F: Functor[F]): Bifunctor[XorT[F, ?, ?]] = {
-    new Bifunctor[XorT[F, ?, ?]] {
-      override def bimap[A, B, C, D](fab: XorT[F, A, B])(f: A => C, g: B => D): XorT[F, C, D] = fab.bimap(f, g)
+  implicit def xorTBifunctor[F[_]](implicit F: Traverse[F]): Bitraverse[XorT[F, ?, ?]] =
```

Comment:
We may want a `Bifunctor` instance for an `XorT` where we have a `Functor[F]` but not a `Traverse[F]` (in fact this comes up quite often with things like `Future`/`Task`). Therefore, I think we should leave this `xorTBifunctor` in place and add a separate `xorTBitraverse` at a separate layer of the `XorTInstances` priority hierarchy (to avoid ambiguous instances).